.circleci/config.base.yml

```diff
@@ -176,6 +176,18 @@ jobs:
           command: |
             yarn verify-api-extract
 
+  verify-yarn-lock:
+    <<: *linux-e2e-executor-large
```

Comment:
nit: is a large executor necessary for this?

Reply:
it seems so. `yarn install` seems to be quite CPU bound.
![image](https://user-images.githubusercontent.com/5849952/217607152-a5cc991d-7535-4faf-bf89-26d3a0f9527a.png)

Reply:
on the other hand, build goes up to 100%, I'll give it a try to decrease size.